src/Sarif.Viewer.VisualStudio/ErrorList/SarifSnapshot.cs

```diff
@@ -188,26 +163,31 @@ private void ErrorListInlineLink_Click(object sender, RoutedEventArgs e)
 
             if (hyperLink != null)
             {
-                Tuple<int, int> indices = hyperLink.Tag as Tuple<int, int>;
+                Tuple<object, int> data = hyperLink.Tag as Tuple<object, int>;
```

Comment:
It's worth a comment that the first element of the tuple is the index of this item in the error list, and the second is the index of the hyperlink target in the result's `RelatedLocations` array.